shared_model/bindings/model_transaction_builder.hpp

```diff
@@ -175,7 +175,7 @@ namespace shared_model {
        */
       ModelTransactionBuilder grantPermission(
           const interface::types::AccountIdType &account_id,
-          const interface::types::PermissionNameType &permission);
+          interface::permissions::Grantable permission);
```

Comment:
const ref is removed for purpose?

Reply:
`Grantable` is just a enum, const does nothing special here